ydb/src/grpc_wrapper/raw_topic_service/common/codecs.rs

```diff
@@ -0,0 +1,36 @@
+use itertools::Itertools;
+use ydb_grpc::ydb_proto::topic::SupportedCodecs;
+
+#[derive(serde::Serialize)]
+pub(crate) enum RawCodec {
+    Unspecified,
+    Raw,
+    Gzip,
+    Lzop,
+    Zstd,
+    Custom(i32),
+}
+
+#[derive(serde::Serialize)]
+pub(crate) struct RawSupportedCodecs {
+    pub codecs: Vec<RawCodec>,
+}
+
+impl From<RawSupportedCodecs> for SupportedCodecs {
+    fn from(value: RawSupportedCodecs) -> Self {
+        Self {
+            codecs: value
+                .codecs
+                .into_iter()
+                .map(|x| match x {
+                    RawCodec::Custom(val) => val,
+                    RawCodec::Unspecified => 0,
+                    RawCodec::Raw => 1,
```

Comment:
Here need to use GRPC constants, becouse it is difficult to match integer constants by mind and very easy add bug by typo.